cluster-up/cluster/k3d-1.25-sriov/provider.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
Same here let's just move this to cluster-up/cluster/k3d/sriov.sh

Reply:
same as https://github.com/kubevirt/kubevirtci/pull/972#discussion_r1129174222